.github/workflows/core.yml

```diff
@@ -30,6 +30,7 @@ jobs:
             --action_env=LD_LIBRARY_PATH \
             --test_env=ENVOY_IP_TEST_VERSIONS=v4only \
             --test_output=all \
+            --define envoy_mobile_listener=disabled \
```

Comment:
Should we flip this in the compile_time_options cc_build CI job?

Reply:
we could as well, but eventually I want to make it the default so I largely wanted it somewhere I could easily regression test and I was also worried about it not playing well with other options.